test/metabase/test/data/dataset_definitions.clj

```diff
@@ -34,3 +34,20 @@
 ;; A small dataset with users and a set of messages between them. Each message has *2* foreign keys to user --
 ;; sender and reciever -- allowing us to test situations where multiple joins for a *single* table should occur.
 (def-database-definition-edn avian-singles)
+
+(def test-data-map
```

Comment:
This one is only used outside of this namespace, but used in conjunction with the field values function below